lib/text/ttml_text_parser.js

```diff
@@ -762,6 +769,16 @@ shaka.text.TtmlTextParser = class {
       if (styleAttributeValue) {
         styleValue = styleAttributeValue;
       }
+
+      styleAttributeValue = XmlUtils.getAttributeNS(
+          inheritedStyles[i],
+          ebuttsNs,
```

Comment:
So, if ebutts and tts are both present, ebutts overrides tts styles?  Is this intended?  Is this likely to occur?

Reply:
It is intentional, the preference could be reversed, but it is not a case that will occur according to the specification. 

https://tech.ebu.ch/docs/tech/tech3380v1_0_1.pdf

Reply:
Okay.  Sounds good to me.

Reply:
I don't know what this is doing, but documents typically exist with both `tts` and `ebutts` prefixes, and the element names do not collide. If the spec namespace URNs are mapped to those prefixes there's no way any `ebutts` style could override a `tts` style.

Reply:
Apologies for the post-merge drive-by comments on Friday, but as I've had a significant hand in both the EBU-TT and TTML specs I'm aware of what "correct" is here. Having taken another look at it, I think what this is doing is, _if_ a style attribute were defined in both the TTML Style and the EBU-TT Style namespace _and_ both were used in the same document, then the EBU-TT Style one would take precedence. This is completely weird from an XML point of view, because in that situation they would be completely different attributes, but from a practical standpoint it can't currently cause any problems.